src/Components/Contact.js

```diff
@@ -0,0 +1,162 @@
+            <div className="contact-form__input-box--text-short">
+              <label for="firstName" className="contact-form__label">
+                <span className="contact-form__mandatory-field" />
+                First Name
+              </label>
+              <input
+                className="contact-form__input-field"
+                aria-required="true"
+                id="firstName"
+                type="text"
+                name="firstName"
+                required
+              />
+            </div>
+            <div className="contact-form__input-box--text-short">
+              <label for="lastName" className="contact-form__label">
+                <span className="contact-form__mandatory-field" />
+                Last Name
+              </label>
+              <input
+                className="contact-form__input-field"
+                aria-required="true"
+                id="lastName"
+                type="text"
+                name="lastName"
+                required
+              />
+            </div>
+            <div className="contact-form__input-box--text-short">
+              <label for="emailAddress" className="contact-form__label">
+                <span className="contact-form__mandatory-field" />
+                Email
+              </label>
+              <input
+                className="contact-form__input-field"
+                aria-required="true"
+                id="emailAddress"
+                type="email"
+                name="emailAddress"
+                required
+              />
+            </div>
+            <div className="contact-form__input-box--text-short">
+              <label for="phoneNumber" className="contact-form__label">
+                <span className="contact-form__mandatory-field" />
+                Phone
+              </label>
+              <input
+                className="contact-form__input-field"
+                aria-required="true"
+                id="phoneNumber"
+                type="tel"
+                name="phoneNumber"
+                required
+              />
+            </div>
+            <div className="contact-form__input-box--text-long">
+              <label for="comment" className="contact-form__label">
+                <span className="contact-form__mandatory-field" />
+                Comment
+              </label>
+              <textarea
+                className="contact-form__textarea"
+                aria-required="true"
+                id="comment"
+                type="text"
+                name="comment"
+                required
+              />
+            </div>
+            <div className="contact-form__input-box--checkbox">
+              <input
+                className="contact-form__checkbox"
+                id="agreement"
+                type="checkbox"
+                name="agreement"
+                required
+              />
+              <span class="contact-form__checkmark" />
```

Comment:
```suggestion
              <span className="contact-form__checkmark" />
```
There are a couple places in this file that need this to be changed.